Remotion/Data/DomainObjects.UnitTests/ClientTransactionFactoryTest.cs

```diff
@@ -74,8 +74,8 @@ public void CreateChildTransaction_WithExtension ()
           .Callback((ClientTransaction transaction) => transaction.Extensions.Add(extensionStub.Object))
           .Verifiable();
 
-      ITransaction rootTransaction = factory.Object.CreateRootTransaction();
-      ITransaction childTransaction = rootTransaction.CreateChild();
+      var rootTransaction = ((ITransactionFactory)factory.Object).CreateRootTransaction();
+      var childTransaction = rootTransaction.CreateChild();
```

Comment:
```suggestion
      ITransaction rootTransaction = ((ITransactionFactory)factory.Object).CreateRootTransaction();
      ITransaction childTransaction = rootTransaction.CreateChild();
```
The interface is a relevant part of the tested semantics.